packages/page-accounts/src/modals/Transfer.tsx

```diff
@@ -208,22 +209,32 @@ function Transfer ({ className = '', onClose, recipientId: propRecipientId, send
         <TxButton
           accountId={propSenderId || senderId}
           icon='paper-plane'
-          isDisabled={!hasAvailable || !(propRecipientId || recipientId) || !amount || !!recipientPhish}
+          isDisabled={!hasAvailable || !(propRecipientId || recipientId) || !amount || !!recipientPhish || !api}
           label={t<string>('Make Transfer')}
           onStart={onClose}
           params={
             canToggleAll && isAll
               ? isFunction(api.tx.balances?.transferAll)
                 ? [propRecipientId || recipientId, false]
                 : [propRecipientId || recipientId, maxTransfer]
-              : [propRecipientId || recipientId, amount]
+              // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+              : !!api && isFunction((api.derive.balances as any).transferAll)
+                ? [propRecipientId || recipientId, amount, api]
+                : [propRecipientId || recipientId, amount]
           }
           tx={
-            canToggleAll && isAll && isFunction(api.tx.balances?.transferAll)
-              ? api.tx.balances?.transferAll
-              : isProtected
-                ? api.tx.balances?.transferKeepAlive
-                : api.tx.balances?.transfer
+            canToggleAll && isAll
+              ? isFunction(api.tx.balances?.transferAll)
+                ? api.tx.balances?.transferAll || transferBalance
+                : isProtected
+                  ? api.tx.balances?.transferKeepAlive || transferBalance
+                  : api.tx.balances?.transfer || transferBalance
+              // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+              : !!api && isFunction((api.derive.balances as any).transferAll)
+                ? transferBalance
+                : isProtected
+                  ? transferBalance
+                  : transferBalance
```

Comment:
Although the `derive` is defined in the `apps-config` for `interBtc`, it's empty when used here. `transferBalance` is what the derive is supposed to be, and it works this way. How can this be correctly implemented using the derive?